lib/module.js

```diff
@@ -2,25 +2,29 @@ const path = require('path')
 const Sentry = require('@sentry/node')
 const WebpackPlugin = require('@sentry/webpack-plugin')
 const deepMerge = require('deepmerge')
+const { Dedupe, ExtraErrorData, RewriteFrames, Transaction } = require('@sentry/integrations')
```

Comment:
The node.js build does not allows any tree shaking. Maybe keeping original reference (`SentryIntegrations = `) would be better idea here so if we make integrations configurable we can use a for loop to enable some.

Reply:
@pi0 if we make integration configurable loop not enough, because integrations can take params, so we will need to pass whole initialized integration instances

Reply:
But then for client integrations we wont be able to serialize them ..

Reply:
We can't do this for client config, maybe it can be in the form of an object that keys are integration name and values are an array that describes params? 

server:

```js
{
  Dedupe: [],
  ExtraErrorData: [],
  RewriteFrames: [],
  Transaction: []
}
```

client:

```js
{
  ExtraErrorData: [],
  ReportingObserver: [],
  RewriteFrames: []
}
```

- Regarding `VueIntegration`, it may have it's own config as we have to also pass Vue instance so need to somehow apply defaults
- Not sure if it is a convention or strict that params should be passed by a single object, but in that case, we may use values as `{}`

Reply:
@pi0 ye, that could work but params could be functions also.

Reply:
We can partially support config or allow function serialization. In both cases supporting _what integration_ to enable is better than nothing :)